pkg/mdm/mdmtest/windows.go

```diff
@@ -466,3 +481,34 @@ func (c *TestWindowsMDMClient) request(path string, reqBody []byte) (*http.Respo
 
 	return response, nil
 }
+
+func (c *TestWindowsMDMClient) getToken() (string, string, error) {
```

Comment:
Tiny nit: this is one rare case where I think named return values help, since it returns two strings it's not as obvious as when it is a single return value + error or even two different values of different types + error. If I want to know what are those strings, I have to search for the return statements in the function and work my way back to what those values are.

Reply:
thanks, 100% agreed!